### ISSUE TITLE
[gamescope-session] Upstream to Fedora

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/rpm-specs-gaming#4]
Attempt to upstream this package into the official Fedora repository.

**Comment by LukeShortCloud** [comment]
Worst-case scenario, we can get this into RPM Fusion short-term.

**Comment by alkazar** [comment]
gamescope-session on its own no longer provides any actual session. It is a base package that other gamescope based sessions rely on.

So we would also need to upstream something like `gamescope-session-steam` for anyone to make any use of it.

**Comment by LukeShortCloud** [comment]
Good point. This report was originally open back when it was one package. We should upstream the two packages (similar to how it is on the AUR): `gamescope-session` for the base package in case others want to BYOS (bring-your-own-session) and `gamescope-session-steam` to get the SteamOS experience with the Steam client.

**Comment by GuestSneezeOSDev** [comment]
why not make a `.spec` for gamescope a `.spec` tells contains the instructions for building the package

**Comment by LukeShortCloud** [comment]
Each folder in this repository contains a related `.spec` file. The `gamescope-session.spec` can be found here:

https://github.com/playtron-os/rpm-specs-gaming/blob/main/gamescope-session/gamescope-session.spec

All of these packages in this repository are hosted on the Fedora Copr. The goal is to get these into Fedora or, at least, RPM Fusion short-term.

https://copr.fedorainfracloud.org/coprs/playtron/gaming/packages/
